charts/thurloe/templates/podDisruptionBudget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Values.name }}-pdb
+spec:
+  # don't reduce serving capacity by more than 30%
+  minAvailable: 30%
```

Comment:
I'm open to changing this number. Since the default replica is 3, I used 30% instead of 1. You can read about using percentages instead of numbers [here](https://v1-20.docs.kubernetes.io/docs/tasks/run-application/configure-pdb/#rounding-logic-when-specifying-percentages)

Reply:
Maybe I'm just reading it wrong but the number seems in conflict with the comment--minimum available being 30% allows a reduction of up to 70%, but the comment says a reduction wouldn't exceed 30%

Reply:
How do you feel about the new comment now?

Reply:
looks great!